app/views/projects/samples/metadata/_edit_form.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= form_with(model: @sample, url: namespace_project_sample_metadata_field_path(sample_id: @sample.id), method: :patch) do |form| %>
+  <%= turbo_frame_tag 'update-alert' %>
+  <div class="grid gap-4">
+  <%= form.fields_for :edit_field do |field| %>
+    <%= field.fields_for :key do |metadata_key| %>
+      <div class="form-field">
+        <%= metadata_key.label t("projects.samples.show.metadata.edit.key"),
+                  class:
+                    "block mb-2 text-sm font-medium text-slate-900 dark:text-white" %>
+        <%= metadata_key.text_field key,
```

Comment:
**suggestion:** Can you try to set autofocus on this when the modal opens

Reply:
Fixed in f4e48c28724d8724dd996660540c38875bf81aef